SYCL/ESIMD/api/functional/ctors/ctor_default.hpp

```diff
@@ -81,27 +81,26 @@ template <typename DataT, typename SizeT, typename TestCaseT> struct run_test {
 
   bool operator()(sycl::queue &queue, const std::string &data_type) {
     bool passed = true;
-    DataT default_val{};
 
+    // We leave this dou to avoiding empty functions optimisation, we do not
```

Comment:
```suggestion
    // We leave this due to avoiding empty functions optimization, we do not
```
or
```suggestion
    // We use it to avoid empty functions being optimized out by compiler
    // We do not check ...
```

Reply:
this comment has been updated in this commit: 43fcd5d89513387f7f9124a2982e43d9b1c34c46